Sources/iPhone/View Controllers/Main/DocumentsViewController_iPhone.m

```diff
@@ -93,6 +93,8 @@ -(void)viewDidLoad {
         self.tblFiles.frame = tmpFrame;
     }
 
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(scrollToTopNotificationHandle) name:EXO_NOTIFICATION_SCROLL_TO_TOP object:nil];
```

Comment:
You need to call `removeObserver:name:object:` in the `dealloc` method of each controller where you observe a notification.